README.md

```diff
@@ -5,18 +5,18 @@
 # GCC Rust
 ![GCC Rust](logo.png?raw=true "GCC rust Logo")
 
-This is a full alternative implementaion of the Rust language ontop of GCC which the goal
+ggcrs is a full alternative implementation of the Rust language ontop of GCC with the goal
 to become fully upstream with the GNU toolchain.
 
 The origin of this project was a community effort several years ago where Rust was still at version 0.9;
 the language was subject to so much change that it became difficult for a community effort to play catch up.
-Now that the language is in a stable state, it is a good time to create alternative compilers. The developers of
-the project are keen “Rustaceans” with a desire to give back to the Rust community and to learn what GCC is capable
+Now that the language is stable, it is an excellent time to create alternative compilers. The developers of
```

Comment:
I personally would have preferred the original wording for this sentence, but strictly speaking I don't think there's anything wrong with the modified one.

Reply:
How about 

> Now that the language builds from the [stable channel](https://blog.datalust.co/our-journey-from-rust-nightly-to->stable/), it is an excellent time to create alternative compilers.

Reply:
Which gives context to what the stable channel is?

Reply:
My understanding of "stable" as originally used was in the general sense of "not changing much anymore", rather than in a specific sense of a "stable channel", for instance. So both the original wording and your original wording would capture that implication, but the suggested change of "stable channel" would not.

Reply:
Ah I see, I tried reading too much into it. 

Reverted the `stable-channel` change to my original edit. I think it makes the wording easier to understand.